rfc-035-use-standard-linting-tools-across-all-our-projects.md

```diff
@@ -0,0 +1,39 @@
+# Use standard linting tools across all of our projects
+
+## Summary
+
+Where available, all new dxw-owned or maintained repositories SHOULD use a
+linter for the programming language the repo is written in, where one is
+available.
+
+## Problem
+
+As developers, we commonly have different styles of writing code. Unchecked this
+can lead to inconsistencies in style, which over time can make code messy, and
+difficult to maintain and review. It can also lead to bikeshedding over code
+style in code reviews, which can cause conflict and slow down code review.
+
+## Proposal
+
+Where available, all new dxw-owned or maintained repositories SHOULD use a
+standard linting tool to make sure code meets a standard style. Examples of this
+include (but are not limited to):
+
+- [Standard.rb](https://github.com/testdouble/standard)
```

Comment:
Is there a reason we have chosen these specific linters over other options?

Reply:
Black (which is an excellent choice) feels like it cleaves the jobs of a linter neatly into two: formatting the code (which it does excellently) and code style advice and enforcement (which it deftly avoids). There may be a role for PyLint or similar.

Reply:
@jdudley1123 I think these are just examples, rather than explicit choices.